sdk/keyvault/keyvault-keys/src/index.ts

```diff
@@ -323,12 +330,17 @@ export class KeysClient {
       delete unflattenedOptions.expires;
       delete unflattenedOptions.requestOptions;
 
+      const span = this.createSpan("createEcKeyMethod", unflattenedOptions);
```

Comment:
This might be a bit of a nit, but why do these all end with "*Method"? It's a hunch, but I'd think we'd want the exact same name so that they can do a search for the symbol and find it.

Reply:
Fixed with https://github.com/Azure/azure-sdk-for-js/pull/4654/commits/64ea0aeda7f3f50e0e6263224a901a85bd2f909e

Reply:
+1, method suffix is unnecessary, should match the symbol.